src/components/CorporateCampaign/CampaignLogoGroup.vue

```diff
@@ -31,6 +31,16 @@ export default {
 		corporateLogoUrl: {
 			type: String,
 			default: ''
+		},
+		logoClasses: {
+			type: String,
+			default: '',
+			required: false
```

Comment:
Doesn't necessarily need to be changed but `required: false` shouldn't be needed on these props

Reply:
I was getting an error when I didn't have this in place stating that the props wasn't defined, even with the default key present...  :/

Reply:
I tried removing all of the `required` props in this changeset locally, and the app built and ran without error. Which error did you encounter?